rust/crates/sift_stream/src/stream/mode/ingestion_config.rs

```diff
@@ -227,15 +229,36 @@ impl SiftStream<IngestionConfigMode> {
             dropped_for_ingestion: false,
         };
 
-        // Send the message for backup first. If this fails, return an error.
+        // Send the message for backup first. If this fails, log an error and continue.
         //
-        // Failure to backup leads to data loss and should be treated as very critical.
-        self.mode
+        // Failure to backup can lead to data loss though it is preferable to attempt
+        // to stream the message to Sift rather than return the error and prevent both.
+        let send_backup_fut = self.mode.stream_system.backup_tx.send(data_msg.clone());
+
+        if let Err(e) = tokio::time::timeout(self.mode.backup_timeout, send_backup_fut).await {
+            #[cfg(feature = "tracing")]
+            tracing::warn!(
+                sift_stream_id = self.mode.sift_stream_id.to_string(),
+                "failed to send data to backup system, data loss may occur: {}",
```

Comment:
nit: interpolation `"failed to send data to backup system, data loss may occur: {e}"`

Reply:
Also we should fix the messaging. `Data loss may occur` isn't totally accurate and may cause panic. Data may have made it to Sift via ingest.

Reply:
Not sending it to the backup manager means that we can't make guarantees about all data being preserved in the event of a reingest **since the last checkpoint.**